CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 8.2.6
```

Comment:
```suggestion
## 8.3.0
```

Since this is a new feature! Also – what's the version in the pubspec?

Reply:
Updated both changelog and pubspec to 8.3.0.